v20.1/cockroach-nodelocal-upload.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
the user must have the admin role, and the node they're uploading to much not have external-io disabled.